packages/manager/modules/pci/src/projects/project/training/jobs/translations/Messages_fr_FR.json

```
@@ -21,18 +21,19 @@
   "pci_projects_project_training_jobs_info_title_3": "Récapitulatif",
   "pci_projects_project_training_jobs_list_submit_success": "Job créé",
   "pci_projects_project_training_jobs_submit_select_image_advanced_tips": "Choississez votre propre image Docker",
-  "pci_projects_project_training_jobs_submit_select_no_data": "Aucun volume de données (dataset) disponible actuellement. Si vous souhaitez ajouter des données pour votre job, il est nécessaire de créer un conteneur de données (Object Storage) et de l'attacher à un volume.",
-  "pci_projects_project_training_jobs_submit_select_no_data_create": "Créer un volume de données",
+  "pci_projects_project_training_jobs_submit_select_no_data": "Aucun conteneur object storage disponible actuellement. Si vous souhaitez ajouter des données pour votre job, il est nécessaire de créer un conteneur de données (Object Storage).",
+  "pci_projects_project_training_jobs_submit_select_no_data_create": "Créer un conteneur object storage",
   "pci_projects_project_training_jobs_submit_select_data": "Attacher des volumes de données au job (facultatif)",
   "pci_projects_project_training_jobs_submit_command_default_select": "Spécifier la commande à lancer. (facultatif)",
   "pci_projects_project_training_jobs_submit_select_command": "Renseigner la commande Docker à lancer dans le conteneur (facultatif)",
   "pci_projects_project_training_jobs_submit_select_command_tips": "Renseigner les arguments (séparés par des espaces) qui seront passés à la commande Docker",
   "pci_projects_project_training_jobs_submit_select_command_tile_title": "Commande spécifique",
   "pci_projects_project_training_jobs_submit_select_command_tile_command": "Arguments",
-  "pci_projects_project_training_jobs_access_url_help": "L'accès HTTP permet d'être redirigé depuis votre navigateur vers le port 80 du conteneur si celui-ci est toujours en cours d'exécution. Pour accéder à l'interface web vous aurrez besoin de vous identifier en tant qu'utilisateur propriétaire du job : {{ owner }}.",
+  "pci_projects_project_training_jobs_access_url_help": "L'accès HTTP permet d'être redirigé depuis votre navigateur vers le port 80 du conteneur si celui-ci est toujours en cours d'exécution. Pour accéder à l'interface web vous aurrez besoin de vous identifier.",
   "pci_projects_project_training_jobs_submit_select_image_tips_help": "Les images hébergées sur un registry privé ou public sont acceptées (Github, GitLab, DockerHub, ...)",
-  "pci_projects_project_training_jobs_submit_attach_source_heading": "Volumes disponibles",
-  "pci_projects_project_training_jobs_submit_attach_target_heading": "Volumes sélectionnés",
-  "pci_projects_project_training_jobs_submit_attach_source_placeholder": "Aucun volume disponible",
-  "pci_projects_project_training_jobs_submit_attach_target_placeholder": "Aucun volume sélectionné"
+  "pci_projects_project_training_jobs_submit_data_container": "Container",
```

```suggestion
  "pci_projects_project_training_jobs_submit_data_container": "Conteneur",
```
to be consistent with https://github.com/ovh/manager/pull/3982/files#diff-2be52894dc9ce9921d8eb885f97f13ffee26682e9d48f119e797d027df106eabR25